### PR TITLE
Adding Karma tests to Travis CI

### PR DESCRIPTION
**Pull request by delapuente**

### CODE REVIEW

**Comment by delapuente** [comment]
This fix #258 
@marco-c, do you mind to review?
And thank you for all the previous assistance.

File: .travis.yml [hunk 1]
```diff
@@ -4,21 +4,30 @@ node_js:
   - '4'
   - 'stable'
 addons:
+  firefox: 'latest-beta'
```
Comment (marco-c) [inline .travis.yml]: Can we use `latest` (44)?

File: .travis.yml [hunk 2]
```diff
@@ -4,21 +4,30 @@ node_js:
...
   apt:
     sources:
     - ubuntu-toolchain-r-test
+    - google-chrome
     packages:
     - g++-4.8
+    - google-chrome-stable
```
Comment (marco-c) [inline .travis.yml:14]: Nit: We could use the `chromium-browser` package so we don't have to add the `google-chrome` PPA earlier.

**Comment by marco-c** [comment]
Nice! Very simple!

**Comment by delapuente** [comment]
@marco-c I don't get it to work with Chromium. Do you mind if we merge this way and see why is not working with Chromium in a follow up?

**Comment by marco-c** [comment]
Sure! It was just a "nit"!

**Pull request merged**